eclipse-collections/src/main/java/org/eclipse/collections/impl/map/ordered/mutable/OrderedMapAdapter.java

```diff
@@ -138,6 +138,12 @@ public int hashCode()
         return OrderedMapAdapter.adapt(new LinkedHashMap<>());
     }
 
+    @Override
+    public MutableOrderedMap<K, V> newEmpty(int initialCapacity)
+    {
+        return OrderedMapAdapter.adapt(new LinkedHashMap<>(initialCapacity));
```

Comment:
I think the implementation for newEmpty() was already wrong, since it ignores the potential comparator.